R/update-input.R

```diff
@@ -37,7 +37,7 @@
 updateTextInput <- function(session = getDefaultReactiveDomain(), inputId, label = NULL, value = NULL, placeholder = NULL) {
   validate_session_object(session)
 
-  message <- dropNulls(list(label=label, value=value, placeholder=placeholder))
+  message <- dropNulls(list(label=processDeps(label, session), value=value, placeholder=placeholder))
```

Comment:
And similar formatting elsewhere, please

```suggestion
  message <- dropNulls(list(
    label=processDeps(label, session), 
    value=value, 
    placeholder=placeholder
  ))
```

Reply:
Sure, I think it's good now.